ui/src/components/Rules/RuleGroup/AddModal/index.tsx

```diff
@@ -580,11 +580,11 @@ const AddModal = (props: AddModal) => {
                           },
                           {
                             id: 1,
-                            name: 'Unmonitor and delete all seasons / episodes',
+                            name: 'Unmonitor show and delete all seasons / episodes',
```

Comment:
Same for this but I need to validate what happens to the episode monitor status. It doesn't look like we explicitly unmonitor each episode, but I am wondering if unmonitoring the season somehow causes the episodes to be unmonitored:

Unmonitors show
Unmonitors seasons
Unmonitors(?) and deletes existing episodes